utilities/constant.py

```diff
@@ -0,0 +1,19 @@
+# MAINNET CONSTANTS
+STARK_CONTRACT_ADDRESS = "0x014F738EAd8Ec6C50BCD456a971F8B84Cd693BBe"
+WEB_PROVIDER = "https://rpc.ankr.com/eth_rinkeby"
+AAVE_MAINNET_CONTRACT_ADDRESS = "AAVE"
+CRUIZE_CONTRACT_ADDRESS = "CRUIZE"
+WALLET_ADDRESS = "0xE0E24a32A7e50Ea1c7881c54bfC1934e9b50B520"
+USDC_ADDRESS = ""
+LINK_ADDRESS = "0x01BE23585060835E02B77ef475b0Cc51aA1e0709"
+WALLET_ADDRESS_2 = "0xBD43b05ac20F96B808d3c644c3CF7add86ad5F58"
+
+# price params
+MAX_WAIT_SECONDS = 60
+SAMPLE_SIZE = 20
```

Comment:
SAMPLE_SIZE - what sample size is it ?

Reply:
it's mean the total number of block from where we will fetch  the gas fee

Reply:
Please rename it to something more appropriate.